app.rb

```diff
@@ -0,0 +1,24 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+
+  configure :development do
+    register Sinatra::Reloader 
+  end
+
+  get '/' do
+    @peep = Peep.all
```

Comment:
Will ask you a couple of questions based on your question around why your logic is not communicating with your view / controller! 

First up, do you get a result from this method call? What do you see when you write p @peep underneath this line, and run your code with rpec?